jupyterlab_nvdashboard/server.py

```diff
@@ -11,7 +11,9 @@
     "/GPU-Utilization": apps.gpu.gpu,
     "/GPU-Memory": apps.gpu.gpu_mem,
     "/GPU-Resources": apps.gpu.gpu_resource_timeline,
-    "/PCI-Throughput": apps.gpu.pci,
+    "/PCIe-Throughput": apps.gpu.pci,
```

Comment:
Do we also want a PCIe timeline?

Reply:
Yes - There should probably be a timeline available for anything volitile.  Perhaps we can open a discussion/issue to solicit feedback on the most desired dashboard options, etc

Reply:
As we are now discussing this in #20 let's ignore this here and get things merged.